apps/concierge_site/lib/helpers/mail_helper.ex

```diff
@@ -47,4 +47,12 @@ defmodule ConciergeSite.Helpers.MailHelper do
   def reset_password_url(reset_token) do
     Helpers.password_reset_url(ConciergeSite.Endpoint, :edit, reset_token)
   end
+
+  @spec track_open_url(%{alert_id: String.t(), notification_id: String.t()}) :: iodata
+  def track_open_url(%{alert_id: alert_id, notification_id: notification_id}) do
+    Helpers.static_url(
+      ConciergeSite.Endpoint,
+      "/notification_email_opened?alert_id=#{alert_id}&notification_id=#{notification_id}"
```

Comment:
Having this defined in the Router generates a `notification_email_opened_url` helper that we should probably use here if we can, instead of manually constructing the path with `static_url`.